### ISSUE TITLE
Sending pdf's to whatsapp groups

### ISSUE THREAD

**Issue by shriharisk7** [Ankit404butfound/PyWhatKit#309]
I have used this library for few of my automating tasks, some of the features did work well like sending images, texts to groups.
However, I would like to have a function that can send pdf's to whatsapp groups like images, whenever I tried to send the pdf I'm only able to send the location of the pdf but not the pdf itself. Similar to sendwhats_image which easily takes in group ID as an argument and works seamlessly requesting for a function that sends pdf's

Thank you

**Comment by VahagnCh** [comment]
Are you able to send images in png? I have been trying to use **pywhatkit.sendwhats_image** but I always get the error (png is not Supported!)

**Comment by shriharisk7** [comment]
> Are you able to send images in png? I have been trying to use **pywhatkit.sendwhats_image** but I always get the error (png is not Supported!)
Yes I'm able to send png images using this method : sendwhats_image(group_id , img_path).

**Comment by DannySye** [comment]
> Are you able to send images in png? I have been trying to use **pywhatkit.sendwhats_image** but I always get the error (png is not Supported!)

It all depends on the platform that you are using, it seems like you are using MacOS, in which case you can only use a jpeg image and is should be below 2mb.
